cmake/dependencies/AMReX.cmake

```diff
@@ -53,6 +53,16 @@ macro(find_amrex)
 
         set(AMReX_SIMD "${WarpX_SIMD}" CACHE INTERNAL "")
 
+        if(WarpX_FASTMATH OR ABLASTR_FASTMATH)
+            set(AMReX_FASTMATH ON CACHE INTERNAL "")
+            # TODO: set consistently (default: ON)
+            # set(AMReX_CUDA_FASTMATH ON CACHE INTERNAL "")
+        else()
+            # TODO: set consistently (default: ON)
+            # set(AMReX_CUDA_FASTMATH ON CACHE INTERNAL "")
+            set(AMReX_FASTMATH OFF CACHE INTERNAL "")
```

Comment:
I thin setting AMReX_FASTMATH to OFF will change the default because it will override AMReX_CUDA_FASTMATH.